src/schema/v2/lot.ts

```diff
@@ -0,0 +1,179 @@
+import { GraphQLError, GraphQLObjectType } from "graphql"
+import { SaleArtworkType } from "schema/v2/sale_artwork"
+import { Response as FetchResponse } from "node-fetch"
+import { ResolverContext } from "types/graphql"
+import {
+  NodeInterface,
+  SlugAndInternalIDFields,
+} from "schema/v2/object_identification"
+import { pageable } from "relay-cursor-paging"
+import { connectionWithCursorInfo } from "./fields/pagination"
+import gql from "lib/gql"
+import { convertConnectionArgsToGravityArgs } from "lib/helpers"
+import { connectionFromArray } from "graphql-relay"
+
+export const LotType = new GraphQLObjectType<any, ResolverContext>({
+  name: "Lot",
+  description:
+    "A lot in an auction containing merged Sale artwork and Lot state data.",
+  interfaces: () => {
```

Comment:
Do you need to fetch a lot via the `node(id: ...` structure? If not, I'd leave this out until needed.

Reply:
Relay generally just needs an `id` field (don't need to specify any interface)